content/metadata.yaml

```diff
@@ -11,6 +11,8 @@ authors:
     initials: JD
     orcid: XXXX-XXXX-XXXX-XXXX
     twitter: johndoe
+    mastodon: johndoe
+    mastodonserver: mastodon.social
```

Comment:
I think it might be preferable to have a single field like `mastodon: johndoe@mastodon.social`. https://bioregistry.io/ doesn't have a `mastodon` record yet, so not sure where to link regarding advice on the proper CURIE format.

Is there a way to resolve a mastodon username@server to direct to the host server?

Aside: IMO the federated model of mastodon leads to a pretty bad user experience. And seems like there will be a lot of tootrot as servers are taken offline. Not against adding links though since there does seem to be adoption among scholars.